PUBLISHING.md

```diff
@@ -1,13 +1,30 @@
 # Publishing Instructions
 
-To update the version number, edit the version in `/components/package.json`. 
+## Automatic Publishing
 
-To publish a new package through NPM, run the following commands from the root folder: 
+This package is published to NPM automatically by CircleCI when code is merged into the `dev` or `master` branches. To publish a new version, simply update the version in `package.json` and merge your code into the appropriate branch.
+- The `dev` branch will publish versions marked as `alpha` or `beta`.
+- The `master` branch will publish any version (`alpha`, `beta`, or `latest`).
+  In both cases, the code will only be published if the version number differs from the current version published under the respective dist tag.
 
+## Manually Publishing
+
+If you need to publish a package manually, you can run the following commands from the root folder:
+
+```sh
+yarn build 
+cd dist
+yarn publish:package
```

Comment:
Is this script accessible when you are in the dist folder?

Reply:
It might be preferable to have the 'cd' commands as part of the publish:package script itself

Reply:
Great catch! This isn't a great situation because the ng-packgr build cmd will strip out the scripts section from the package.json.  I'll have to figure out the best way to fix this.

Reply:
I think it makes more sense to move the publish command out from the components folder, make it top level and move the cd command to the publish cmd like you mentioned.